packages/core/core-flows/src/cart/workflows/add-shipping-method-to-cart.ts

```diff
@@ -83,13 +107,17 @@ export const addShippingMethodToWorkflow = createWorkflow(
             )
           }
 
+          const methodData = data.validatedMethodData?.find((methodData) => {
+            return methodData?.[option.id]
+          })
```

Comment:
q: If this bit is not valid, should we return early or throw than trying to proceed?

Reply:
If the fulfillment provider doesn't return anything (maybe there is no validation), then I'd rather proceed and default to an empty object than throw